language-plutus-core/test/Evaluation/Constant/Success.hs

```diff
@@ -145,17 +143,8 @@ test_applyBuiltinNameSuccess =
         , test_typedSHA3Success
         ]
 
--- | Generates in-bounds constants and checks that their evaluation results in an 'EvaluationSuccess'.
-test_evalInBounds :: TestTree
-test_evalInBounds =
```

Comment:
Given that we don't have sizes anymore, that was a pretty useless test.

Reply:
:+1: